reshape/subset/config.yaml

```diff
@@ -0,0 +1,5 @@
+access:
+  policies:
+    - type: Query
+      policyDefault:
```

Comment:
Is this file needed as it takes the default of exposing all fields? Or did you put it in here on purpose to reduce confusion for users trying out the other reshape examples?

Reply:
It's needed as for all the examples I made the statement that they would be accessible without authorization. Without this file then authorization is needed.